tests/EventStream.Tests/EventDispatcherTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using EventStream.Tests.Fakes;
+using FluentAssertions;
+using Microsoft.Framework.DependencyInjection;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace EventStream.Tests
+{
+    public class EventDispatcherTests
+    {
+        static IServiceProvider CreateProvider() 
+                                    => new ServiceCollection().AddInMemoryEventStream().BuildServiceProvider();
```

Comment:
Is this the indentation VS put in for you? I would expect that when this was all one line, hitting enter with the cursor infront of `=>`, that it would be a single tab in from `static`. Make sense?